src/hotspot/share/opto/vectornode.hpp

```diff
@@ -1627,6 +1627,22 @@ class VectorUCastS2XNode : public VectorCastNode {
   virtual int Opcode() const;
 };
 
+class HF2FVNode : public VectorNode {
```

Comment:
You may use same naming convention as used for other vector casting IR nodes 
VectorCastH2F and F2H